app/services/s3/storeConsentFgrToS3Bucket.ts

```diff
@@ -0,0 +1,51 @@
+import { PutObjectCommand } from "@aws-sdk/client-s3";
+import { config } from "~/services/env/env.server";
+import { today } from "~/util/date";
+import { createClientDataConsentFgr } from "./createClientDataConsentFgr";
+import { sendSentryMessage } from "../logging";
+import { getSessionIdByFlowId } from "../session.server";
+
+const createConsentDataBuffer = (sessionId: string, request: Request) => {
```

Comment:
this is just my thought; sometimes, we pass the whole request object, which can be unnecessary. but i am fine as is

```suggestion
const createConsentDataBuffer = (sessionId: string, { headers }: { headers: Headers }) => {
```

Reply:
same idea, but why not simply `sessionId: string , headers: Headers`?